x-pack/plugins/task_manager/server/task_running/ephemeral_task_runner.ts

```diff
@@ -206,9 +210,15 @@ export class EphemeralTaskManagerRunner implements TaskRunner {
       );
     }
     this.logger.debug(`Running ephemeral task ${this}`);
-    const apmTrans = apm.startTransaction(this.taskType, 'taskManager ephemeral run', {
-      childOf: this.instance.task.traceparent,
-    });
+    const apmTrans = apm.startTransaction(
```

Comment:
This is analogous to the transaction in task runner `run` function, where the transaction type is `task-run` and the transaction name is the task type, with no entity id, so for consistency, should this be `apm.startTransaction(this.taskType, 'task-run-ephemeral')` with no entity id?

Reply:
@ymao1 do you think that it might be more useful to do something like this?

| transaction.type  | transaction.name | labels.ephemeral
| ------------- | ------------- | ------------- | 
| task-run  | task 1 | false
| task-run  | task 2 | false
| task-run  | task 3 | true

Reply:
yeah, I think that makes sense!